documentation/modules/configuring/con-maintenance-time-window-definition.adoc

```diff
@@ -6,13 +6,22 @@
 = Scheduling maintenance time windows
 
 [role="_abstract"]
-Schedule Cluster Operator certificate renewals for Kafka clusters to minimize impact on client applications.
-Use time windows in conjunction with the xref:con-certificate-renewal-str[renewal periods of the CA certificates created by the Cluster Operator] (`Kafka.spec.clusterCa.renewalDays` and `Kafka.spec.clientsCa.renewalDays`).
+Use maintenance time windows to control when the Cluster Operator performs certificate renewals and rolling restarts.
+This helps minimize disruption to Kafka clients by scheduling updates at convenient times.
 
-Updates are usually triggered by changes to the `Kafka` resource by the user or through user tooling.
-Rolling restarts for certificate expiration may occur without `Kafka` resource changes.
-While unscheduled restarts shouldn't affect service availability, they could impact the performance of client applications.
-Maintenance time windows allow scheduling of these updates for convenient times.
+Maintenance time windows apply only to certificates generated automatically by the Cluster Operator.
+They do not apply to externally managed certificates, such as custom listener certificates or certificates issued by external certificate managers.
+As a result, using custom certificates that are not managed by the Cluster Operator may lead to restarts occurring outside the defined maintenance windows.
```

Comment:
I'm not sure if this is really clear / as simple as described.

IIRC, The maintenance time window does not apply when a custom CA is used, and the custom CA is updated and restart will be triggered even outside the window.

But it would be applied to the time-based renewal of the server certificates in the same fashion as it is to the Strimzi-managed CA, as that is not user-triggered action.

For listener certificates, the update happens through user-action. So the maintenance time windows are never used.